aiida_restapi/models.py

```diff
@@ -2,23 +2,30 @@
 """Schemas for AiiDA REST API"""
 # pylint: disable=too-few-public-methods
 
-import typing
+from typing import Optional
+
+from aiida.orm import User as AiidaUser
 from pydantic import BaseModel, Field
 
 
 class User(BaseModel):
     """AiiDA User model."""
-    id: typing.Optional[int] = Field(description='Unique user id (pk)')
-    first_name: typing.Optional[str] = Field(
-        description='First name of the user')
-    last_name: typing.Optional[str] = Field(
-        description='Last name of the user')
-    institution: typing.Optional[str] = Field(
-        description='Host institution or workplace of the user')
-    email: str = Field(description='Email address of the user')
+
+    id: Optional[int] = Field(description="Unique user id (pk)")
+    first_name: Optional[str] = Field(description="First name of the user")
+    last_name: Optional[str] = Field(description="Last name of the user")
+    institution: Optional[str] = Field(
+        description="Host institution or workplace of the user"
+    )
+    email: str = Field(description="Email address of the user")
+
+    class Config:
+        """The models configuration."""
+
+        orm_mode = True
```

Comment:
FYI @ltalirz @NinadBhat, according to https://pydantic-docs.helpmanual.io/usage/models/#orm-mode-aka-arbitrary-class-instances, I had to add this (other wise you get mypy errors `error: "User" does not have orm_mode=True  [pydantic-orm]`)

Reply:
Thanks! Looking at the link you provided, I think you can actually delete our (re-)implementation of `from_orm`.

I just tested it locally and it seemed to work - can you include that here?

Reply:
done